### PR TITLE
[#12081] Fix remaining accessibility tests

### PR DESCRIPTION
**Pull request by zhaojj2209**
Part of #12081

Fixes all remaining accessibility tests so as to prepare the `user-friendliness` branch for merging into `master`. The commits are split by component/page for easier review.

Note that this does not mean all accessibility issues on TEAMMATES are fixed; accessibility tests can only catch a portion of accessibility issues and the rest will still need to be identified and fixed manually.

### CODE REVIEW

File: src/web/app/pages-instructor/instructor-course-student-edit-page/instructor-course-student-edit-page.component.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
             </div>
           </div>
           <div class="row mb-3">
-            <label class="col-sm-2 col-form-label">Section Name:</label>
+            <label for="sectionname" class="col-sm-2 col-form-label">Section Name:</label>
```
Comment (jasonqiu212) [inline src/web/app/pages-instructor/instructor-course-student-edit-page/instructor-course-student-edit-page.component.html:35]: Should we be consistent with previous `label`s and use `for="section-name"`?

Comment (jasonqiu212) [reply src/web/app/pages-instructor/instructor-course-student-edit-page/instructor-course-student-edit-page.component.html:35]: I notice that this is following the `id` of the input below. Should we modify the input's `id`s to use `-`?

**Review by jasonqiu212 — COMMENTED** [review]
LGTM!

File: src/web/app/pages-instructor/instructor-course-student-edit-page/instructor-course-student-edit-page.component.html (hunk 1 above)
Comment (zhaojj2209) [reply src/web/app/pages-instructor/instructor-course-student-edit-page/instructor-course-student-edit-page.component.html:35]: Good catch, will modify the IDs here.

**Pull request merged**